src/query/on_conflict.rs

```diff
@@ -20,10 +20,17 @@ pub enum OnConflictTarget {
 pub enum OnConflictAction {
     /// Do nothing
     DoNothing,
+    /// Update column value of existing row
+    Update(Vec<UpdateStrat>),
+}
+
+/// Represents strategies to update column in ON CONFLICT (upsert) actions
+#[derive(Debug, Clone)]
+pub enum UpdateStrat {
     /// Update column value of existing row with inserting value
-    UpdateColumns(Vec<DynIden>),
+    Column(DynIden),
     /// Update column value of existing row with expression
-    UpdateExprs(Vec<(DynIden, SimpleExpr)>),
+    Expr(DynIden, SimpleExpr),
 }
```

Comment:
I got a suggested naming for `UpdateStrat`, maybe `OnConflictUpdate`?

Reply:
I have changed the name to `OnConflictUpdate`